HCK09_2020_Remote/README.md

```diff
@@ -0,0 +1,79 @@
+
+<img alt="Remote hackathon" src="../HCK08_2020_Remote/logo_brain_text_white_hor.png">
+
+## Welcome to the 8th NWB User Days Workshop!
+
+  * [Dates and Location](#dates-and-location)
+  * [Registration](#registration)
+  * [Logistics](#logistics)
+  * [Organizing Committee](#organizing-committee)
+  * [Resources](#resources)
+  * [How to prepare?](#how-to-prepare)
+  * [Objective](#objective)
+  * [Projects](#projects)
+     * [Creating a New Project](projects/README.md)
+  * [Agenda](#agenda)
+
+## Dates and Location
+
+- **Dates:** September, 2020
+- **Location:** Your home!
+
+## Registration
+
+The workshop is open to the public. Please complete our [registration form](https://forms.gle/8LYAx743D4rEwSkT7) to attend the event. Anyone interested in NWB may request an invitation. 
+
```

Comment:
Need to make a new registration form and update the link before merging.

Reply:
already did

Reply:
but I guess the link isn't updated? Hold on, I'll fix this